public/app/plugins/datasource/prometheus/metric_find_query.test.ts

```diff
@@ -142,7 +142,7 @@ describe('PrometheusMetricFindQuery', () => {
       expect(fetchMock).toHaveBeenCalledWith({
         method: 'GET',
         url:
-          'proxied/api/v1/series?match%5B%5D=metric%7Blabel1%3D%22foo%22%2C+label2%3D%22bar%22%2C+label3%3D%22baz%22%7D&start=1524650400&end=1524654000',
+          'proxied/api/v1/series?match%5B%5D=metric%7Blabel1%3D%22foo%22%2C%20label2%3D%22bar%22%2C%20label3%3D%22baz%22%7D&start=1524650400&end=1524654000',
```

Comment:
These decode to the exactly same thing:
![image](https://user-images.githubusercontent.com/30407135/116059872-9c683880-a681-11eb-8acd-2dc3bc5d745e.png)
![image](https://user-images.githubusercontent.com/30407135/116059891-a25e1980-a681-11eb-9772-d776a648c0ee.png)